container/functional/keylime_ipv6_multihost/payload-script/local_action_modify_payload.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+MY_UUID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+if [ ! -f "$1" ]; then
+    echo "Input JSON file \"$1\" not found"
+    exit 1
+fi
+
+TYPE=$(grep -o "\"type\":[ ]*\"\([^\"]*\)\"" "$1" | sed -e 's/\"type\":[ ]*\"\([^\"]*\)\"/\1/g')
+
+if [ "$TYPE" != "revocation" ]; then
+    echo "Input JSON type is not revocation"
+    exit 1
+fi
+
+EVENT_UUID=$(grep -o "\"agent_id\":[ ]*\"\([^\"]*\)\"" "$1" | sed -e 's/\"agent_id\":[ ]*\"\([^\"]*\)\"/\1/g')
```

Comment:
Just to confirm, you are not using `jq` because it may not be available in the keylime image, am I right?

Reply:
Right, the goal was to require very minimum dependencies to run.